tests/integrate_test/iiss/prevote/test_prep.py

```diff
@@ -633,3 +633,147 @@ def check_query_via_icx_sendtransaction(self, method: str, params: dict, expecte
                                        func_name=method,
                                        params=params)
         return self.process_confirm_block_tx([tx], expected_status=expected_status)
+
+    def test_get_preps_normal(self):
```

Comment:
I want to add additional test-cases
1. startRanking or endRanking are less than zero
2. startRanking is larger than # of PReps

Reply:
append test case.

pass case
* normal
* over endRanking than prep count

else:
* reverse
* overflow
* zero
* negative